docs/upgrade.html

```diff
@@ -20,6 +20,28 @@
 <script id="upgrade-template" type="text/x-handlebars-template">
 
 <h5><a id="upgrade_280_notable" href="#upgrade_280_notable">Notable changes in 2.8.0</a></h5>
+<ul>
+    <li>
+        The 2.8.0 release added a new Authorizer interface introduced in
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-679%3A+Producer+will+enable+the+strongest+delivery+guarantee+by+default">KIP-679</a>.
+        Custom authorizer implementations should consider overriding this default implementation because:
+        <ul>
+            <li>
+                1. The default implementation iterates all AclBindings multiple times, without any caching
+                   by principal, host, operation, permission types, and resource types. More efficient
+                   implementations may be added in custom authorizers that directly access cached entries.
```

Comment:
Should be sufficient to provide a summary: e.g. `Custom authorizers should consider providing a more efficient implementation that supports audit logging and any custom configs or access rules`.

Reply:
Yes, refer to the Authorizer javadocs for detailed reasoning.